routes/web.php

```diff
@@ -109,7 +110,9 @@ function () {
 
             // ======== Calendar routes ========
             Route::get('calendar', [CalendarController::class, 'index'])->name('calendar');
-
+            Route::get('/roles', [RolesController::class, 'index']);
+            Route::post('/roles', [RolesController::class, 'store']);
+            Route::get('/allRoles', [RolesController::class, 'getRoles']);
```

Comment:
use slug naming for routing e.g `all-roles`

Reply:
Ok, I ll be updating these changes.